### DISCUSSION TITLE
Suggestions on Installing Imagick

### DISCUSSION THREAD

**Discussion by camber800** [leokhoa/laragon#424] [Q&A] [answered]
What is the best way to install Imagick? I'm just getting started working with Laragon and need to install Imagick on php 8.1.10. I found this https://www.youtube.com/watch?v=t48wd1mctP0 but after following the instructions I could not activate Imagick in the Laragon PHP Extensions list.  I'm on Windows 10, Laragon 6.0.220916.

Thank you!

**Comment by Pen-y-Fan** [comment, marked as answer]
The version of Imagick needs to match the version of PHP e.g. 8.1.10 TS x64.

See my blog on how to pick the correct version of PHP, which will help to pick the correct match of Imagick:

- [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/#selecting-the-correct-version) #Selecting the correct version

If laragon doesn't automatically detect Imagick, it may need to be manually added the extension to **php.ini**.

```ini
[imagick]
extension=imagick
```

See the Xdebug or Opcache section of the above blog post for an example and details on how to access php.ini.

**Reply by camber800** [reply]
The missing piece was the extension=imagick

I was unable to activate Imagick via the Laragon UI in PHP > Extensions for some reason. Once I added the line in php.ini it is working.

Thank you very much for the response and the installation doc.

**Reply by 3xploiton3** [reply]
its  seem doesnt  work with wordpress 6.3.1
![image](https://github.com/leokhoa/laragon/assets/19517680/281ec23c-c4fa-4b0b-b851-356d1ef3f233)
![image](https://github.com/leokhoa/laragon/assets/19517680/c19cdc51-0d39-46e2-9d6e-1cf5f26d573e)

**Comment by smultr0n** [comment]
This page guide goes over the installation process quite nicely
https://mlocati.github.io/articles/php-windows-imagick.html

**Reply by camber800** [reply]
Thank you-much appreciated.

**Reply by isaiasxavier** [reply]
That perfectly worked, thank you!

**Reply by kindiaing** [reply]
非常棒，链接已收藏！

**Comment by 3xploiton3** [comment]
https://pecl.php.net/package/imagick
